policyengine_uk/variables/gov/hmrc/study_childcare_entitlement/care_to_learn_eligible.py

```diff
@@ -0,0 +1,41 @@
+from policyengine_uk.model_api import *
+
+
+class care_to_learn_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible for Care to Learn childcare support"
+    documentation = "Whether the person is eligible for Care to Learn support"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        # Only parents can be eligible, not children
+        is_parent = ~person("is_child", period)
```

Comment:
Then use `~is_child` below. `~is_child` does not imply `is_parent`.
```suggestion
        is_child = person("is_child", period)
```